src/mongo/commands.ts

```diff
@@ -30,7 +31,21 @@ export class MongoCommands {
 				throw new Error('Please connect to the database first');
 			}
 			const result = await database.executeCommand(command);
```

Comment:
Don't run executeCommand for a 'find'. Just run it in the node itself as a part of getChildren

Reply:
I'll have to run `executeCommand` for a `findOne` too. Excluding `executeCommand` for a find alone seems clunky, given that the statements around the node creation+getChildren are the same for the `find` and `findOne` case. 

That said, I'm performing this change, have a look in the next iteration.